h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -409,14 +409,21 @@ def test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_mode
 
 def test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_runtime_wins():
     ds = import_dataset()
-    aml = H2OAutoML(project_name="py_all_stopping_constraints", seed=1, max_models=20, max_runtime_secs=12)
+    aml_probe = H2OAutoML(project_name="py_all_stopping_constraints_probe",
+                          max_models=5,
+                          seed=1, exclude_algos=["XGBoost", "StackedEnsemble"])
```

Comment:
why is your probe using less models?
I would understand if `max_models` was the same in both cases, but here I don't see the connection between `t_probe` and `t_tested`...

Also, excluding `StackedEnsemble` definitely makes sense given that SEs are allowed to use extra-time after `max_runtime_secs`, but why do you want to exclude `XGBoost`?